fastlane/lib/fastlane/actions/clear_derived_data.rb

```diff
@@ -8,6 +10,15 @@ def self.run(options)
         UI.success("Successfully cleared Derived Data ♻️")
       end
 
+      # Helper Methods
+      def self.xcode_preferences
+        file = File.expand_path("~/Library/Preferences/com.apple.dt.Xcode.plist")
+        if File.exist?(file)
+          return CFPropertyList.native_types(CFPropertyList::List.new(file: file).value)
```

Comment:
Can we also check to make sure that `CFPropertyList::List.new(file: file).value` exists before calling `CFPropertyList.native_types`?